sdcm/cluster.py

```diff
@@ -4876,7 +4876,7 @@ def reconfigure_scylla_monitoring(self):
             for db_node in self.targets["db_cluster"].nodes:
                 monitoring_targets.append(f"[{getattr(db_node, attr_name)}]:9180")
             monitoring_targets = " ".join(monitoring_targets)
-            if self.params.get("ip_ssh_connections") == "ipv6":
+            if self.params.get("ip_ssh_connections") != "ipv6":
                 monitoring_targets = monitoring_targets.replace("[", "").replace("]", "")
```

Comment:
Why all this logic ? We have a function to identify if it's a ipv6 or not. And append the brackets as needed.
Why putting them it, and then taking them out ?

Reply:
The change of logic was implemented here: https://github.com/scylladb/scylla-cluster-tests/commit/741691d991981935db321dca324dce147369b29f
I am just fixing what I changed here: https://github.com/scylladb/scylla-cluster-tests/commit/e60144ee
If it is not really required, I don't mind.